src/app/client/src/app/modules/learn/learn-routing.module.ts

```diff
@@ -20,6 +21,9 @@ const routes: Routes = [
       telemetry: { env: telemetryEnv, pageid: 'learn', type: 'view' }
     }
   },
+  {
+    path: 'redirect', component: RedirectComponent
```

Comment:
why do we need "redirect" route under learn routes? this component is used from home, learn, library and anonymous consumption pages also.

Reply:
Initially it was implemented for courses, so we have it under learn,